main.js

```diff
@@ -0,0 +1,28 @@
+// Remove the "All Posts" link from the header.
+document.querySelector("header nav ul li").remove()
+
+//  Remove the span element that contains the "Read Time" in the featured post. The featured post is the first post on the page.
+let els2 = document.querySelectorAll(".featured span");
+
+for(let el2 of els2){
+    if(el2.textContent.includes("Read Time")){
+        el2.remove();
+    }
+}
+
+//  Remove the last post from the page, titled "Stop Planning".
+let els3 = document.querySelectorAll("main .posts article");
+
+for(let el3 of els3){
+    if(el3.textContent.includes("You -- yes you!")){
+        el3.remove();
+    }
+}
```

Comment:
Interesting way to do this one! I like that it will always target the "Stop Planning" article no matter its position in the DOM. 
Here's another way to do this if removing the **_last element_** specifically is the main priority:
`els3[els3.length - 1].remove()`